mod/caching/spec/set/type_plus_right/wikirate_company/source_spec.rb

```diff
@@ -0,0 +1,13 @@
+# encoding: UTF-8
+
+describe Card::Set::TypePlusRight::WikirateCompany::Source do
+  it 'updated cached count' do
+    samsung_sources = Card.fetch "Samsung+source"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.